<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateInputDialogClass</class>
 <widget class="QDialog" name="DateInputDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter date</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0,0">
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1,0,1">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="dateLabel">
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="todaySelect">
        <property name="text">
         <string>&amp;Today</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="yesterdaySelect">
        <property name="text">
         <string>&amp;Yesterday</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="otherDateSelect">
        <property name="text">
         <string>Other &amp;date:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDateEdit" name="otherDateInput">
        <property name="displayFormat">
         <string>MM/dd/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="dateRangeSelect">
        <property name="text">
         <string>Date &amp;range:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDateEdit" name="firstDateInput">
        <property name="displayFormat">
         <string>MM/dd/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="dateRangeToLabel">
        <property name="text">
         <string>to</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QDateEdit" name="lastDateInput">
        <property name="displayFormat">
         <string>MM/dd/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DateInputDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DateInputDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>otherDateSelect</sender>
   <signal>toggled(bool)</signal>
   <receiver>otherDateInput</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateRangeSelect</sender>
   <signal>toggled(bool)</signal>
   <receiver>firstDateInput</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateRangeSelect</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateRangeToLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateRangeSelect</sender>
   <signal>toggled(bool)</signal>
   <receiver>lastDateInput</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
